server/odc-service/src/main/java/com/oceanbase/odc/service/resultset/ResultSetExportTask.java

```diff
@@ -266,6 +276,21 @@ private void setColumnConfig(DataTransferConfig config, ResultSetExportTaskParam
         }
     }
 
+    private String getTimeoutSettingStatement(DialectType dialectType) {
+        switch (dialectType) {
+            case OB_MYSQL:
+            case OB_ORACLE:
+            case ODP_SHARDING_OB_MYSQL:
+                return "set ob_query_timeout = " + DEFAULT_TIMEOUT_SECONDS * 1000000L;
```

Comment:
we may not set this timeout settings by a static field, shall we add this parameter to the `ResultSetExportTaskParameter`?

we can give it a default value cause the front end does not set it~

Reply:
sure, I added a parameter in `ResultSetExportTaskParameter `